create_bfb

```diff
@@ -257,9 +257,9 @@ copy_rpm()
 }
 
 # Copy the content of RPM packages
-for package in  mlxbf-bfscripts util-linux dosfstools e2fsprogs kmod glibc coreutils pciutils usbutils chkconfig mft mft-oem kernel-mft tar which curl openssl $ADDON_RPMS
+for package in  mlxbf-bfscripts util-linux dosfstools e2fsprogs kmod glibc coreutils pciutils usbutils chkconfig mft mft-oem kernel-mft tar which curl openssl nvme-cli mmc-utils $ADDON_RPMS
 do
-	copy_rpm $package
+	copy_rpm $package || true
```

Comment:
Is it okay for `copy_rpm` to fail? Won't that mask errors?

Reply:
I checked this with the Nvidia team, and seems that is added to prevent errors in case of already existing packages etc, but nothing critical a prt of copying package